src/index.html

```diff
@@ -17,16 +17,40 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <body>
+      <div class="container">
+        <form
+          class="search-bar input__big"
+          data-qa="big"
+        >
+          <input
+            class="input input__big search-bar__input"
```

Comment:
that's strange you have same class `input__big` in `form` and in `input`

Remove it from the `form` element. Add new class, like `search-bar__big` if you need to apply some styles to big/small form. But actually, you don't need here styles from `input__big`